net-libs/libssh/libssh-0.8.4.recipe

```diff
@@ -108,6 +124,12 @@ INSTALL()
 	packageEntries devel \
 		"$developDir" \
 		"$libDir"/cmake
+
+	if [ -z "$secondaryArchSuffix" ]; then
+		install -d -m 755 "$developDocDir"
+		mv -T doc/html "$developDocDir"
```

Comment:
I'd replace  `mv -T doc/html "$developDocDir"` by `cp -r doc/html/* "$developDocDir"` because using `mv` makes the **second** and following invocations of `hp libssh` fail until `/boot/home/haikuports/net-libs/libssh/work-0.8.4/flag.build` is deleted or `hp -c libssh` is called:
```
...
-- Installing: /packages/libssh-0.8.4-1/.self/lib/libssh.so
/packages/libssh_devel-0.8.4-1/.self
mv: cannot stat 'doc/html': No such file or directory
```

BTW, I don't know if this would be an improvement, but if we wanted to have the html files installed in `develop/documentation/libssh/html/` instead of `develop/documentation/libssh/` then, of course, we should use `cp -r doc/html "$developDocDir"` instead of `cp -r doc/html/* "$developDocDir"`.